src/utils/registerNewNetwork.ts

```diff
@@ -31,3 +108,17 @@ export const registerNewNetwork = async (
 
   return registerCustomArbitrumNetwork(arbitrumNetwork);
 };
+
+export async function registerNewNetworkFromParentPublicClient<TChain extends Chain | undefined>({
```

Comment:
I don't see much value in this light wrapper tbh, I think we're good with just the other one